docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,48 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each column of data impacts the performance of the model.
```

Comment:
>column of data [](start = 93, length = 14)

"feature" rather than "column of data". The end features in the model might not be exactly the input columns.